linkcheck/checker/mailtourl.py

```diff
@@ -145,57 +152,110 @@ def check_email_syntax(self, mail):
         # http://www.rfc-editor.org/errata_search.php?eid=1003
         if len(mail) > 256:
-            self.set_result(_("Mail address `%(addr)s' too long. Allowed 256 chars, was %(length)d chars.") % \
-            {"addr": mail, "length": len(mail)}, valid=False, overwrite=False)
+            self.set_result(
+                _(
+                    "Mail address `%(addr)s' too long. Allowed 256 chars, "
+                    "was %(length)d chars."
+                )
+                % {"addr": mail, "length": len(mail)},
+                valid=False,
+                overwrite=False,
+            )
             return
         if "@" not in mail:
-            self.set_result(_("Missing `@' in mail address `%(addr)s'.") % \
-            {"addr": mail}, valid=False, overwrite=False)
+            self.set_result(
+                _("Missing `@' in mail address `%(addr)s'.") % {"addr": mail},
+                valid=False,
+                overwrite=False,
+            )
             return
         # note: be sure to use rsplit since "@" can occur in local part
         local, domain = mail.rsplit("@", 1)
         if not local:
-            self.set_result(_("Missing local part of mail address `%(addr)s'.") % \
-            {"addr": mail}, valid=False, overwrite=False)
+            self.set_result(
+                _("Missing local part of mail address `%(addr)s'.") % {"addr": mail},
+                valid=False,
+                overwrite=False,
+            )
             return
         if not domain:
-            self.set_result(_("Missing domain part of mail address `%(addr)s'.") % \
-            {"addr": mail}, valid=False, overwrite=False)
+            self.set_result(
+                _("Missing domain part of mail address `%(addr)s'.") % {"addr": mail},
+                valid=False,
+                overwrite=False,
+            )
             return
         if len(local) > 64:
-            self.set_result(_("Local part of mail address `%(addr)s' too long. Allowed 64 chars, was %(length)d chars.") % \
-            {"addr": mail, "length": len(local)}, valid=False, overwrite=False)
+            self.set_result(
+                _(
+                    "Local part of mail address `%(addr)s' too long. "
+                    "Allowed 64 chars, was %(length)d chars."
+                )
+                % {"addr": mail, "length": len(local)},
+                valid=False,
+                overwrite=False,
+            )
             return
         if len(domain) > 255:
-            self.set_result(_("Domain part of mail address `%(addr)s' too long. Allowed 255 chars, was %(length)d chars.") % \
-            {"addr": mail, "length": len(local)}, valid=False, overwrite=False)
+            self.set_result(
+                _(
+                    "Domain part of mail address `%(addr)s' too long. "
+                    "Allowed 255 chars, was %(length)d chars."
+                )
+                % {"addr": mail, "length": len(local)},
```

Comment:
(it's not a fluke, they're all like that.  something else to get used to?)

Reply:
> Going through my "another PR" comments and extracting them into separate issues is going to be a job...

I have a cunning plan which is to create one issue that points to this PR and then resolve the comments here. If you get a chance to resolve any comments which are just asides that would be great (I have read them all). So far I have just resolved actual changes.

Reply:
https://github.com/linkchecker/linkchecker/issues/439